README.md

```diff
@@ -1,53 +1,129 @@
 # Discord bot
 
-# !!! Currently developing version 2.0 with rewritting code to C# !!!
+[![Docker image build](https://github.com/seker212/Discord_bot/actions/workflows/container-image-update.yml/badge.svg?branch=master-2.0)](https://github.com/seker212/Discord_bot/actions/workflows/container-image-update.yml)
 
-[![Chi-chan bot Container](https://github.com/seker212/Discord_bot/actions/workflows/Chi-chan_Container.yml/badge.svg?branch=master&event=push)](https://github.com/seker212/Discord_bot/actions/workflows/Chi-chan_Container.yml)
+## Table of contents
 
-## Running locally
+1. [Running bot](#tc1)
+    1. [Important informations](#tc11)
+    2. [Doker run](#tc13)
+        1. [Enviroment informations](#tc131)
+        2. [Using docker cli](#tc132)
+        3. [Using docker compose](#tc133)
+    3. [Local run](#tc12)
+2. [Debug and development](#tc2)
+    1. [Database schemas](#tc21)
 
-Clone this repo with checkout to branch `master-2.0`. Run the `DiscordBot` project. :bangbang: Before running remember to add to the main direcotry file `token.txt` with discord bot token.
+## Running bot <a id="tc1"></a>
 
-## Running in docker
+This part is about how to run bot depending on enviroment used.
 
-### Using docker compose
+### Imporatnt informations <a id="tc11"></a>
 
-Container can be run using provided docker compose file provided example provided in `docker-compose-template.yaml`, or from the example below.
+Before trying to run you need to aquire token from [Discord](https://discord.com/developers/applications).
+
+Project supports use of [`Seq`](https://datalust.co/seq) server which can be run locally or remotly for storing logs.
+
+### Docker run <a id="tc13"></a>
+
+Here steps for use of deployable version of DiscordBot. Default Docker Hub image could be found [here](https://hub.docker.com/r/seker212/discord_bot)
+
+#### Enviroment informations <a id="tc131"></a>
+
+Running in docker requires setting important evnviromental variables as well as some volumes for files that should be accessible after shutdown of container.
+
+Variables marked with `Seq server` are referenced to different image for running Seq toghether with DiscordBot
```

Comment:
Seq has it's own documentation, and it's a good one. It's not our responsibility to explain seq docker image or duplicate it's docs in any way. Just leave an url to it's documentation page